src/component/Form.js

```diff
@@ -24,7 +24,6 @@ export default function Form() {
     };
     document.querySelector('.title').value = '';
     document.querySelector('.author').value = '';
```

Comment:
**Optional**
- When the book is being added successfully, please implement the possibility to reset the form's inputs' fields to return empty. 👍